packages/gamepads_web/lib/gamepads_web.dart

```diff
@@ -0,0 +1,137 @@
+// ignore_for_file: avoid_dynamic_calls, omit_local_variable_types
+
+import 'dart:async';
+import 'dart:js_interop';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+import 'package:gamepads_platform_interface/api/gamepad_controller.dart';
+import 'package:gamepads_platform_interface/api/gamepad_event.dart';
+import 'package:gamepads_platform_interface/gamepads_platform_interface.dart';
+
+import 'package:gamepads_web/gamepad_detector.dart';
+import 'package:web/web.dart' as web;
+
+class GamePadState {
```

Comment:
Since there already is a `GamepadState` this should be named something else, and possibly be made private.